### PR TITLE
supply int64_t as the dumb library's custom offset type

### PR DESCRIPTION
**Pull request by fabiangreffrath**

### CODE REVIEW

File: prboom2/src/MUSIC/dumbplayer.c [hunk 1]
```diff
@@ -68,10 +68,9 @@ const music_player_t db_player =
 #if !defined(_FILE_OFFSET_BITS) || (_FILE_OFFSET_BITS < 64)
 #ifdef _MSC_VER
 #define DUMB_OFF_T_CUSTOM __int64
-#elif defined(__APPLE__)
-#define DUMB_OFF_T_CUSTOM off_t
 #else
-#define DUMB_OFF_T_CUSTOM off64_t
+#include <stdint.h>
+#define DUMB_OFF_T_CUSTOM int64_t
```
Comment (fabiangreffrath) [inline prboom2/src/MUSIC/dumbplayer.c:73]: Alternative: `long long`

**Comment by kraflab** [comment]
@BigBigos could you confirm this works fine for you?

File: prboom2/src/MUSIC/dumbplayer.c (hunk 1 above)
Comment (kraflab) [reply prboom2/src/MUSIC/dumbplayer.c:73]: I prefer `int64_t` for sure :^)

Comment (fabiangreffrath) [reply prboom2/src/MUSIC/dumbplayer.c:73]: 👍

**Comment by BigBigos** [comment]
This builds correctly on both 32-bit and 64-bit Linux, thanks!

**Comment by kraflab** [comment]
Thanks 👌

**Pull request merged**